__tests__/app.test.js

```diff
@@ -299,3 +299,58 @@ describe('POST /api/articles/:article_id/comments', () => {
     
 });
 
+describe('PATCH /api/articles/:article_id', () => {
+    it('responds with updated article', () => {
+        const articleId = 1;
+        const incrementVotes = 5;
+        let initialVotes;
+      
+        return request(app)
+          .get(`/api/articles/${articleId}`)
+          .expect(200)
+          .then((res) => {
+            initialVotes = res.body.article.votes;
```

Comment:
Im not quite sure what this bit it doing - you should just be able to do a patch request to the route, like another dev would do